### ISSUE TITLE
Maximum Call Stack Size Exceeded error on save

### ISSUE THREAD

**Issue by philrenaud** [sveltejs/vite-plugin-svelte#40]
## Description

We're migrating our sapper codebase over to sveltekit, and everything's going smoothly, except on save of .svelte files.

On initial run / render, everything is fine and our app works as expected. However, we are encountering a `RangeError: Maximum call stack size exceeded` on save of any .svelte file:

Here's the log from `npm run dev`: 

```
  SvelteKit v1.0.0-next.95

  local:   http://localhost:3000
  network: not exposed

  Use --host to expose server to other devices on this network


10:57:52 a.m. [vite] new dependencies found: mousetrap, updating...
10:57:52 a.m. [vite] ✨ dependencies updated, reloading page...
10:57:54 a.m. [vite] new dependencies found: js-cookie, updating...
10:57:54 a.m. [vite] ✨ dependencies updated, reloading page...
10:57:56 a.m. [vite] new dependencies found: lodash, amplitude-js, updating...
10:57:56 a.m. [vite] ✨ dependencies updated, reloading page...
Exception in PromiseRejectCallback:
/Users/philrenaud/www/dashboard/node_modules/vite/dist/node/chunks/dep-994e0558.js:69268
    });
    ^

RangeError: Maximum call stack size exceeded

```

A colleague (@sdwvit) has noticed that if you add the highlighted text below to index.js in this plugin, the error no longer occurs:
![image](https://user-images.githubusercontent.com/713991/117024604-25a5ed80-acc8-11eb-961b-8c880737fcb7.png)

## Additional Information

Our repo is ~300 routes and components. I can provide a list of dependencies if that would be helpful.

- The output of `npx envinfo --system --npmPackages vite,svelte,svelte-hmr,@sveltejs/vite-plugin-svelte --binaries --browsers`
```
➜ npx envinfo --system --npmPackages vite,svelte,svelte-hmr,@sveltejs/vite-plugin-svelte --binaries --browsers
npx: installed 1 in 1.384s

  System:
    OS: macOS 10.15.7
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 113.50 MB / 16.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 14.7.0 - ~/.nvm/versions/node/v14.7.0/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 6.14.7 - ~/.nvm/versions/node/v14.7.0/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Browsers:
    Brave Browser: 84.1.12.112
    Chrome: 90.0.4430.93
    Firefox: 88.0
    Safari: 14.0.3
  npmPackages:
    svelte: ^3.35.0 => 3.37.0 
    vite: ^2.2.3 => 2.2.3 
```

**Comment by dominikg** [comment]
Hi, please add the version of `svelte-preprocess` you are using and a reproduction repository that demonstrates whats causing this. 

There has been an issue that was fixed recently regarding scss:  https://github.com/sveltejs/svelte-preprocess/issues/346  
If there are other instances, we should try to fix those too.

cc @kaisermann

**Comment by AaronDDM** [comment]
We were using version `4.5.2`. Looks like this is fixed in `4.7.3` - will just continue testing, but if that's the case this issue can be closed!

**Comment by philrenaud** [comment]
Seems like we're good here -- closing issue! If anybody else runs into this in the future, upgrade `svelte-preprocess` as recommended above.
